Formula/minetest.rb

```diff
@@ -49,12 +59,19 @@ def install
     args << "-DBUILD_CLIENT=1" << "-DBUILD_SERVER=0"
     args << "-DENABLE_FREETYPE=1" << "-DCMAKE_EXE_LINKER_FLAGS='-L#{Formula["freetype"].opt_lib}'"
     args << "-DENABLE_GETTEXT=1" << "-DCUSTOM_GETTEXT_PATH=#{Formula["gettext"].opt_prefix}"
-    args << "-DIRRLICHT_LIBRARY=#{Formula["irrlicht"].opt_frameworks}/IrrFramework.framework"
```

Comment:
Is this not needed?

Reply:
That statement causes a build failure on Linux.
As other platforms work fine without the statement, I removed that.

Reply:
How does this affect linkage with `irrlicht`?

Reply:
Do you mean how the build fails on Linux?

On Linux, the directory corresponding to `Formula["irrlicht"].opt_frameworks` is not generated and the compiler cannot find headers and link library.

Reply:
No, I'm asking how the removal of this flag affects linkage with `irrlicht` on macOS.

Reply:
I didn't consider that.
Now I checked outputs of `brew linkage` with and without the statement.

<details>
<summary>With the statement:</summary>

```
System libraries:
  /usr/lib/libSystem.B.dylib
  /usr/lib/libbz2.1.0.dylib
  /usr/lib/libc++.1.dylib
  /usr/lib/libcurl.4.dylib
  /usr/lib/libform.5.4.dylib
  /usr/lib/libiconv.2.dylib
  /usr/lib/libncurses.5.4.dylib
  /usr/lib/libobjc.A.dylib
  /usr/lib/libsqlite3.dylib
  /usr/lib/libz.1.dylib
Variable-referenced libraries:
  @executable_path/../Frameworks/IrrFramework.framework/Versions/A/IrrFramework
  @executable_path/../Frameworks/libfreetype.6.dylib
  @executable_path/../Frameworks/libgmp.10.dylib
  @executable_path/../Frameworks/libjpeg.9.dylib
  @executable_path/../Frameworks/libluajit-5.1.2.dylib
  @executable_path/../Frameworks/libogg.0.dylib
  @executable_path/../Frameworks/libopenal.1.dylib
  @executable_path/../Frameworks/libpng16.16.dylib
  @executable_path/../Frameworks/libvorbis.0.dylib
  @executable_path/../Frameworks/libvorbisfile.3.dylib
Dependencies with no linkage:
  irrlicht
  libogg
  libvorbis
```
</details>

<details>
<summary>Without the statement:</summary>

```
System libraries:
  /usr/lib/libSystem.B.dylib
  /usr/lib/libbz2.1.0.dylib
  /usr/lib/libc++.1.dylib
  /usr/lib/libcurl.4.dylib
  /usr/lib/libform.5.4.dylib
  /usr/lib/libiconv.2.dylib
  /usr/lib/libncurses.5.4.dylib
  /usr/lib/libobjc.A.dylib
  /usr/lib/libsqlite3.dylib
  /usr/lib/libz.1.dylib
Variable-referenced libraries:
  @executable_path/../Frameworks/libfreetype.6.dylib
  @executable_path/../Frameworks/libgmp.10.dylib
  @executable_path/../Frameworks/libjpeg.9.dylib
  @executable_path/../Frameworks/libluajit-5.1.2.dylib
  @executable_path/../Frameworks/libogg.0.dylib
  @executable_path/../Frameworks/libopenal.1.dylib
  @executable_path/../Frameworks/libpng16.16.dylib
  @executable_path/../Frameworks/libvorbis.0.dylib
  @executable_path/../Frameworks/libvorbisfile.3.dylib
Dependencies with no linkage:
  irrlicht
  libogg
  libvorbis
```
</details>

Diff:
```diff
@@ -10,7 +10,6 @@
   /usr/lib/libsqlite3.dylib
   /usr/lib/libz.1.dylib
 Variable-referenced libraries:
-  @executable_path/../Frameworks/IrrFramework.framework/Versions/A/IrrFramework
   @executable_path/../Frameworks/libfreetype.6.dylib
   @executable_path/../Frameworks/libgmp.10.dylib
   @executable_path/../Frameworks/libjpeg.9.dylib
```

There is a difference in linkage indeed, though somewhat it doesn't affect the test.